Pulumiup 2024 first article (#4244)
pulumi/pulumi-hugo: themes/default/content/blog/announcing-pulumiup-conference-2024/index.md

---
title: "Announcing PulumiUP Conference 2024"

# The date represents the post's publish date, and by default corresponds with
# the date and time this file was generated. Dates are used for display and
# ordering purposes only; they have no effect on whether or when a post is
# published. To influence the ordering of posts published on the same date, use
# the time portion of the date value; posts are sorted in descending order by
# date/time.
date: 2024-05-01T17:27:25Z

# The draft setting determines whether a post is published. Set it to true if
# you want to be able to merge the post without publishing it.
draft: false

# Use the meta_desc property to provide a brief summary (one or two sentences)
# of the content of the post, which is useful for targeting search results or
# social-media previews. This field is required or the build will fail the
# linter test. Max length is 160 characters.

> meta_desc:

Author: ```suggestion
meta_desc: PulumiUP 2024 is back! Register to attend on September 18, and dive into the future of cloud, Infrastructure as Code (IaC), AI, platform engineering, and more.
```

# The meta_image appears in social-media previews and on the blog home page. A
# placeholder image representing the recommended format, dimensions and aspect
# ratio has been provided for you.

> meta_image: meta.png

Author: ```suggestion
meta_image: pulumiup-2024-announcement.png
```

# At least one author is required. The values in this list correspond with the
# `id` properties of the team member files at /data/team/team. Create a file for
# yourself if you don't already have one.
authors:
    - sara-huddleston

# At least one tag is required. Lowercase, hyphen-delimited is recommended.
tags:
    - pulumi-events

# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md
# for details, and please remove these comments before submitting for review.
---

Today we announce our annual PulumiUP virtual conference taking place on September 18, 2024. PulumiUP is our global cloud and IaC conference for anyone interested in infrastructure as code, cloud, AI, platform engineering, DevOps, and cloud culture. Every year, we strive to build a program packed with technical talks, demos, and best practices to empower you with new learnings and inspiration that you can apply in your professional life. Attend to hear from industry leaders, experts, and community peers.

<!--more-->

## What is PulumiUP?

> PulumiUP isn't just another conference, it's a global gathering of minds passionate about cloud technologies, AI, DevOps, and everything in between. Whether you're an industry veteran or just dipping your toes into the vast ocean of cloud computing, PulumiUP has something for everyone. 

Reviewer: ```suggestion
PulumiUP isn't just another conference, it's a global gathering of minds passionate about cloud technologies, AI, DevOps, and everything in between. Whether you're an industry veteran or just dipping your toes into the vast ocean of cloud computing, PulumiUP has something for everyone.
```

> You will hear keynote from Pulumi leaders, Founder/CEO Joe Duffy and CTO Luke Hoban, and from industry leaders on their experience in cloud adoption, innovation and transformation and how they are leverage Pulumi to make it happen. We will have special industry expert speakers to help you navigate the everchanging world of the cloud and hear from the community, which can be you if you have a story to tell that brings value to cloud ecosystem community.

Reviewer 2: This feels like it could use a little tightening up. Maybe something like this?

```suggestion
You'll hear keynotes from Pulumi Founder/CEO Joe Duffy and CTO Luke Hoban, stories from industry leaders on their experiences with cloud adoption and transformation (and how they used Pulumi to make it happen), and talks from other members of the Pulumi community (like you!) aimed at helping you navigate the ever-changing world of the cloud.
```

> You will hear keynote from Pulumi leaders, Founder/CEO Joe Duffy and CTO Luke Hoban, and from industry leaders on their experience in cloud adoption, innovation and transformation and how they are leverage Pulumi to make it happen. We will have special industry expert speakers to help you navigate the everchanging world of the cloud and hear from the community, which can be you if you have a story to tell that brings value to cloud ecosystem community. 

Reviewer: ```suggestion
You will hear keynote from Pulumi leaders, Founder/CEO Joe Duffy and CTO Luke Hoban, and from industry leaders on their experience in cloud adoption, innovation and transformation and how they are leverage Pulumi to make it happen. We will have special industry expert speakers to help you navigate the everchanging world of the cloud and hear from the community, which can be you if you have a story to tell that brings value to cloud ecosystem community.
```

## Call for Speakers: Your Story Matters

One exciting addition to this year's event is the call for speakers. We invite you, the community, to share your stories, experiences, and expertise with fellow attendees. Whether it's a deep dive into a groundbreaking project, lessons learned from failures, or innovative best practices, your voice matters at PulumiUP.

### Key Dates to Remember

- CFP opens: Thursday, April 25, 2024.
- CFP closes: Monday, June 17, 2024.
- CFP Notifications: Monday, July 22, 2024.
  
### Selection Criteria

> At PulumiUP, diversity and originality are celebrated. We welcome proposals from all corners of the tech community and prioritize fresh perspectives and solutions. Regardless on where you are in the world, we want to hear from you. Whether you're a seasoned speaker or a first-timer, we encourage you to submit your ideas. Just remember, no vendor pitches allowed!

Reviewer 2: ```suggestion
At PulumiUP, diversity and originality are celebrated. We welcome proposals from all corners of the tech community and prioritize fresh perspectives and solutions. Regardless of where you are in the world, we want to hear from you. Whether you're a seasoned speaker or a first-timer, we encourage you to submit your ideas. Just remember, no vendor pitches allowed!
```

### Talk Formats and Themes

You can submit up to two proposals in either the Lightning Talk (15-minute) or traditional session (30-minute) formats. The conference will cover a wide array of themes, including:

> - **Infrastructure as Code**: From cloud providers to kubernetes, explore the latest trends and best practices in cloud computing and IaC.

Reviewer 2: ```suggestion
- **Infrastructure as Code**: From cloud providers to Kubernetes, explore the latest trends and best practices in cloud computing and IaC.
```

> - **Platform Engineering & DevOps**: Learn how to streamline CI/CD practices, automate infrastructure operations, build internal developer portals (IDP), and enhance developer experiences.

Reviewer 2: ```suggestion
- **Platform Engineering & DevOps**: Learn how to streamline CI/CD practices, automate infrastructure operations, build internal developer portals (IDP), and improve developer experiences.
```

> - **AI / ML**: Dive into the world of AI and ML, covering topics like deploying AI application, LLM models, MLOps, and AIOps.

Reviewer 2: ```suggestion
- **AI / ML**: Dive into the world of AI and ML with topics like deploying AI application, LLM models, MLOps, and AIOps.
```
